### ISSUE TITLE
FlowCrypt compose screen covers Gmail draft editor

### ISSUE THREAD

**Issue by DenBond7** [FlowCrypt/flowcrypt-browser#4690]
When a user opens some regular draft and after that opens FlowCrypt compose screen the last one covers the Gmail draft editor and doesn't allow editing the draft.

Steps to reproduce:

1. Open some draft
2. Click on Secure Compose
3. We will receive the following screen

![image](https://user-images.githubusercontent.com/2863246/190399589-1ad13692-9189-4442-8ccb-c78c543ac2b3.png)

4. User is not able to edit the draft

_Suggestion:_ we can move FlowCrypt compose screen as Gmail does it

![image](https://user-images.githubusercontent.com/2863246/190399794-100a834e-6c8c-4518-97ff-3cc7743ed12c.png)

**Comment by tomholub** [comment]
Right. This is not just for drafts, generally compose windows stack over one another independently, causing what you observed.

I've been getting around it by minimizing one of the windows if it's in the way. But you're right it's not ideal.
